TestingApp/ViewController.swift

```diff
@@ -172,6 +180,27 @@ extension ViewController {
         }
     }
 
+    func requestVisitorCode() {
+        Glia.sharedInstance.requestVisitorCode { visitorCode, error in
+            if let error = error {
+                print("Error getting visitor code.", error.localizedDescription)
+            } else if let visitorCode = visitorCode {
+                print("Visitor code is:", visitorCode)
```

Comment:
be aware that @EgorovEI has introduced a new interface and you will need to update this code in the future.

Reply:
Now the work has been merged, so the new method can be integrated. However, I don't know how to handle the scenario, because this task won't be able to be merged because Bitrise tests will fail because the new method hasn't been released yet.

Reply:
Maybe it makes sense to use feature flag for this case?

Reply:
Can someone provide help with that issue? I have currently provided the changes from @EgorovEI's PR but I am not aware how to bypass any bitrise issues.